.circleci/config.yml

```diff
@@ -23,8 +23,8 @@ orbs:
                 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y -t x86_64-unknown-linux-musl
                 echo "export PATH=\"$HOME/.cargo/bin:$PATH\"" >> $BASH_ENV
           - run:
-              name: "Install cargo-make"
-              command: cargo install cargo-make
```

Comment:
The readme also has a reference to cargo make:
https://github.com/heroku/buildpacks-jvm/blame/768dda084984d3dcb9218fabcf889e1659f32fdb/buildpacks/jvm-function-invoker/README.md#L6

Also the release workflow:
https://github.com/heroku/buildpacks-jvm/blob/773f34196abe8322acfcd7d21fea73253aa6ba38/.github/workflows/release.yml#L39-L44